src/main/resources/camunda/add_defendant_litigation_friend.bpmn

```diff
@@ -79,87 +79,130 @@
     <bpmn:sequenceFlow id="Flow_RPA_NO_CONTINUOUS_FEED" name="No RPA Continuous feed" sourceRef="Gateway_RPA_CONTINUOUS_FEED" targetRef="Activity_1tqimyo">
       <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${!empty flowFlags.RPA_CONTINUOUS_FEED &amp;&amp; !flowFlags.RPA_CONTINUOUS_FEED}</bpmn:conditionExpression>
     </bpmn:sequenceFlow>
+    <bpmn:serviceTask id="LitigationFriendAddedNotifyRespondentSolicitor2" name="Notify respondent solicitor 2" camunda:type="external" camunda:topic="processCaseEvent">
```

Comment:
Can we add a test for a 1v2 Two Solicitor scenario to ensure that when the flag is enabled, we take this path

Reply:
Added tests :)